app/scripts/react.module.ts

```diff
@@ -4,17 +4,25 @@ import * as ReactGA from 'react-ga';
 
 import {SETTINGS} from './modules/core/config/settings';
 
-import {AuthenticationServiceInject} from './modules/core/authentication/authentication.service';
+// angular services to make importable
+import {AUTHENTICATION_SERVICE, AuthenticationServiceInject} from './modules/core/authentication/authentication.service';
 
-export const REACT_MODULE = 'spinnaker.react';
+// react component wrappers around angular components
+import {ButtonBusyIndicatorInject} from './modules/core/forms/buttonBusyIndicator/ButtonBusyIndicator';
 
 // Initialize React Google Analytics
 if (SETTINGS.analytics.ga) {
   ReactGA.initialize(SETTINGS.analytics.ga, {});
 }
 
+export const REACT_MODULE = 'spinnaker.react';
 module(REACT_MODULE, [
   'bcherny/ngimport',
+  AUTHENTICATION_SERVICE,
 ]).run(function ($injector: any) {
-    AuthenticationServiceInject($injector);
+  // Make angular services importable
+  AuthenticationServiceInject($injector);
+
+  // Convert angular components to react
+  ButtonBusyIndicatorInject($injector);
```

Comment:
I'm not sure if it's better to consolidate the instantiation of services here in a `run` block vs. doing the same thing in the component itself. It's a little less code, maybe, but I can imagine this is going to get unwieldy quickly.

Does it make more sense to handle this in the module itself? e.g.
```
module(BUTTON_BUSY_INDICATOR_COMPONENT, [])
  .component('buttonBusyIndicator', new ButtonBusyIndicatorComponent());
  .run(($injector: any) => BusyButtonIndicatorInject($injector));
```
(or whatever it might look like)